script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_9B.txt

```diff
@@ -1,15 +1,15 @@
 <Page1>
 A block of ice flies through the air.
-The ice splits into spears that go straight for my head.
-―――Should I avoid them or take them head-on?
-Do I continue to hide it or do I cast away my only weapon?
+The ice forms a spear, a bullet fired to shoot off my head.//氷は槍と化し、弾丸となって俺の頭を撃ち抜きにくる。
```

Comment:
I think pierce my head is better.
If not, then 'bullet fired to blow my brain out'